packages/core/src/geometry/Geometry.ts

```diff
@@ -199,7 +199,8 @@ export class Geometry
             buffer = new Buffer(buffer);
         }
 
-        buffer.index = true;
+        buffer.type = BUFFER_TYPE.ELEMENT_ARRAY_BUFFER;
```

Comment:
Refactoring the `index` property to `type` is a breaking change. Since we're getting v6 out, you're lucky @GoodBoyDigital :P

Reply:
i got your back bro, `index` still exists on `Buffer` for backwards compatibility 👍